packages/playback-core/src/index.ts

```diff
@@ -182,9 +182,11 @@ export const teardown = (mediaEl?: HTMLMediaElement | null, hls?: Pick<Hls, 'det
   }
   if (mediaEl?.mux && !mediaEl.mux.deleted) {
     mediaEl.mux.destroy();
-    mediaEl.mux;
+    delete mediaEl.mux;
```

Comment:
Should mux-embed do this for us?

Reply:
Looks like it already does and I just have trust issues. 😄

Reply:
Actually on closer inspection this only happens when a new "monitor" is setup. I think we should leave this as is for now and follow up w/exploring "deleting on destroy" in mux-embed (not sure if there are Good Reasons™️ for not doing that).